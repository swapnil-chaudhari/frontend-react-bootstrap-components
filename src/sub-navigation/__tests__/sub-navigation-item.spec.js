import React from 'react';
import renderShallow from 'render-shallow';
import noop from 'src/utils/noop';
import SubNavigationItem from '../sub-navigation-item';

describe('<SubNavigationItem>', () => {
    describe('when it renders', () => {
        let component;

        beforeAll(() => {
            component = renderShallow(
                <SubNavigationItem
                    href="#"
                    className="active"
                    onClick={ noop }
                >Project #1</SubNavigationItem>
            ).output;
        });

        test('should have proper markup', () => {
            expect(component).toMatchSnapshot();
        });
    });
});
