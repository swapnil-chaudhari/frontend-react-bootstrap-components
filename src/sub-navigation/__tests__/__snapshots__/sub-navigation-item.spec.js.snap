// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubNavigationItem> when it renders should have proper markup 1`] = `
<a
  className="active"
  href="#"
  onClick={[Function]}
>
  Project #1
</a>
`;
